test/mirk_convergence_tests.jl

```diff
@@ -1,6 +1,3 @@
-using Base.Test
-using BoundaryValueDiffEq, DiffEqBase, DiffEqDevTools
-
 # First order test
```

Comment:
As a style thing, I prefer to have the `using` statements at the top of the test to make the script runnable. It's not necessary for `Pkg.test` since you'll `using` in `runtests` most of the stuff you need, but I like the idea that each test script is a runnable file that you can use to reproduce examples (i.e. every test file is also an example for how to use the package). Not mandatory, but something to keep in mind.